### DISCUSSION TITLE
Noob with Stepper Motor Problems

### DISCUSSION THREAD

**Discussion by JCowper** [arkypita/LaserGRBL#963] [Q&A]
Hello,

I am not having any success getting the Stepper motors to move with the Laser GRBL program. Any suggestions would be appreciated. 

I have also taken a picture of my wiring job to the CNC Shield on my Arduino Uno.

![15873271292888953156554848717368](https://user-images.githubusercontent.com/61848538/79698697-95317200-8258-11ea-954e-9cf5b158b93d.jpg)

**Comment by StuartB4** [comment]
You missed a pin on the motor connectors on the cnc shield. There's one left empty. Move the plugs down 1 pin.


![nmnmm](https://user-images.githubusercontent.com/16980026/79705668-69d97200-82ae-11ea-90f1-cff40772832b.jpg)

**Comment by JCowper** [comment]
Thanks, but no change to the problem.

**Comment by JCowper** [comment]
I know Laser GRBL is sending GCode to the board because the laser is turning on and off. Unfortunately the motors are not moving at all. They are not even attempting to move at all.

**Comment by arkypita** [comment]
Are you sure of the orientation of the stepper drivers? Mine have a serigraphy that says where the EN pin is, so I have no difficulty, but I don't see any indication on yours.

**Comment by gmmanonymus111** [comment]
If that is a drv8825 driver, it is most likely been put together all right. (the EN tag is printed on both the shield and the unederside of the driver)

Could you check that blue 12V input socket on your shield? On mine, the soldering broke off when I tightened the screws. It is barely visible, but when you try to turn the screw, the little pin (on the downside of the shield) turns as well. If that's the case, you only need to resolder it.

EDIT: If that's not the case, please check your drivers individually with the help of [this](https://www.makerguides.com/drv8825-stepper-motor-driver-arduino-tutorial/) tutorial

**Comment by Schwarzi07** [comment]
I had the same problem, but adjusting the current limiter Potentiometer made the problem go away. Have you also checked that you are using the correct voltage for your stepper motor
